Libraries/Components/Slider/Slider.js

```diff
@@ -115,6 +116,24 @@ var Slider = React.createClass({
     thumbImage: Image.propTypes.source,
 
     /**
+     * Color of the foreground switch grip.
+     * @platform android
+     */
+    thumbTintColor: ColorPropType,
+
+    /**
+     * Color of the slider track.
+     * @platform android
+     */
+    trackTintColor: ColorPropType,
```

Comment:
@janicduplessis Hmm... do they specify the progress and and track background colors? the names are really misleading. I thought that they have something to do with minimum and maximum values.

IMO we should deprecate those and use better prop names.